### PR TITLE
feat(edgeless): edgeless copilot actions

### PR DESCRIPTION
**Pull request by doouding**
continue https://github.com/toeverything/blocksuite/pull/6620

### Add new actions
- findActions
- brainstormMindmap
- expandMindmap
- brainstormMindmap
- makeItReal
- createSlides
- explain

### CODE REVIEW

**Comment by doouding** [comment]
@pengx17 I listed some new added actions in the PR description.

File: packages/presets/src/ai/config/edgeless-copilot.ts [hunk 1]
```diff
@@ -1,317 +1,172 @@
...
-          },
-        ],
-        actions: [],
-      },
-      errorStateConfig: {
-        upgrade: () => {},
-        responses: [],
-      },
-    };
+import type { AIItemGroupConfig } from '@blocksuite/blocks';
```
Comment (pengx17) [inline packages/presets/src/ai/config/edgeless-copilot.ts:1]: I think @Flrande mentioned that the edgless config should be located at `ai/entries/edgeless`

Comment (Flrande) [reply packages/presets/src/ai/config/edgeless-copilot.ts:1]: We can move item components from blocks to preset

**Review by Saul-Mirone — APPROVED** [review]

File: packages/presets/src/ai/config/edgeless-copilot.ts (hunk 1 above)
Comment (Flrande) [reply packages/presets/src/ai/config/edgeless-copilot.ts]: Entries subfolders correspond to setup functions for each ui entry (such as listening for events to trigger panel when a space is entered)

**Review by Flrande — APPROVED** [review]

**Pull request merged**